### ISSUE TITLE
Add app shortcut to QR code scanner

### ISSUE THREAD

**Issue by fynngodau** [corona-warn-app/cwa-wishlist#460]
### Current Implementation

There is a shortcut to the contact journal as suggested in #295.

### Suggested Enhancement

There should also be a shortcut to the QR code scanner for the new check-in functionality.

### Expected Benefits

Users could open the scanner with lightning speed :zap:

**Comment by Ein-Tim** [comment]
This is actually already implemented and will come with CWA 2.0: https://github.com/corona-warn-app/cwa-app-ios/pull/2235

**Comment by fynngodau** [comment]
@Ein-Tim Are you certain? I also find this neither by trying it out nor by looking for it in the source code (specifically, I checked [`AppShortcuts.kt`](https://github.com/corona-warn-app/cwa-app-android/blob/release/2.0.x/Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/AppShortcuts.kt) and [`AppShortcutsHelper.kt`](https://github.com/corona-warn-app/cwa-app-android/blob/release/2.0.x/Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/shortcuts/AppShortcutsHelper.kt)). Hence, I opened this issue.

**Comment by Ein-Tim** [comment]
@fynngodau 

I was quite certain that I saw a PR regarding this during one of my _visits_ in the Android repo. But in fact, I can't find it anymore now.
If you checked these files and there's nothing than it seems like this was only implemented for iOS...
Sorry for the confusion 😕

This should probably be moved to the Android repo, or?

**Comment by fynngodau** [comment]
Yes, granted the context that this is already implemented for iOS, it would be helpful if one of the community managers could move this to the Android repository. Would you like to do this @dsarkar?
